ddtrace/contrib/internal/langgraph/patch.py

```diff
@@ -139,6 +140,9 @@ def _stream():
                 break
             except Exception:
                 span.set_exc_info(*sys.exc_info())
+                integration.llmobs_set_tags(
```

Comment:
is this part of the langchain feature or is this a separate fix?

Reply:
separate fix, for making sure we submit our llmobs tags on error

Reply:
if you choose not to pull this out into its own pull request, i'd recommend including a `fix` line in the release note of this pull request, because doing so will ensure we don't lose track of the fact that this change contained both a feature and a fix.